<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qflex</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/ico/qflex.png</normaloff>:/ico/qflex.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollLayout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>82</width>
         <height>399</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="labelsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionMettre_jour"/>
    <addaction name="action_Exporter"/>
    <addaction name="actionT_l_charger_tout_les_documents"/>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Navigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="action_Tout_r_duire"/>
    <addaction name="action_Suivant"/>
    <addaction name="action_Pr_c_dant"/>
   </widget>
   <widget class="QMenu" name="menu_Affichage">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
    <addaction name="action_Plein_cran"/>
   </widget>
   <widget class="QMenu" name="menu_Option">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="actionModifier_les_sources"/>
    <addaction name="actionParam_tres"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Affichage"/>
   <addaction name="menu_Navigation"/>
   <addaction name="menu_Option"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMettre_jour">
   <property name="text">
    <string>Mettre à &amp;jour...</string>
   </property>
  </action>
  <action name="action_Suivant">
   <property name="text">
    <string>&amp;Suivant</string>
   </property>
  </action>
  <action name="action_Pr_c_dant">
   <property name="text">
    <string>&amp;Précédant</string>
   </property>
  </action>
  <action name="action_Plein_cran">
   <property name="text">
    <string>&amp;Plein écran</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="action_Tout_r_duire">
   <property name="text">
    <string>&amp;Tout réduire</string>
   </property>
  </action>
  <action name="actionT_l_charger_tout_les_documents">
   <property name="text">
    <string>Télécharger tout les documents</string>
   </property>
  </action>
  <action name="action_Exporter">
   <property name="text">
    <string>&amp;Exporter...</string>
   </property>
  </action>
  <action name="actionModifier_les_sources">
   <property name="text">
    <string>Modifier les &amp;sources...</string>
   </property>
  </action>
  <action name="actionParam_tres">
   <property name="text">
    <string>Paramètres...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
